contracts/README.md

```
@@ -1,12 +1,18 @@
 # Proof-of-Humanity v2 - Contracts
 
+### Integration guidance
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)